corehq/apps/userreports/expressions/specs.py

```diff
@@ -625,6 +634,92 @@ def __str__(self):
                                   str(self._value_expression))
 
 
+def _get_user(domain, doc_id):
```

Comment:
can we not use the same serialization as the APIs

Reply:
The resulting doc has significant differences from the current doc, and the change breaks existing UCRs.

To use the same serialization as the APIs, we would need to add data to the API serialization, change existing data source definitions, and rebuild the data sources.